packages/glide-authenticate/src/lib.ts

```diff
@@ -1,60 +0,0 @@
-import { Connection, OAuth2 } from "jsforce";
```

Comment:
This was split up into multiple files to make it easier to maintain in the future. Each file in the `events` directory now represents a single lambda event handler.